src/main/java/fruit/shop/service/StringConnector.java

```diff
@@ -0,0 +1,8 @@
+package fruit.shop.service;
+
+import java.util.Map;
+
+public interface StringConnector {
+    String getStringReport(Map<String, Integer> records);

```

Comment:
Use more specific names
```suggestion
public interface ReportService {
    String generateReport();
```